front/src/styles/scss/applications/operationalStudies/_scenario.scss

```
@@ -683,14 +679,18 @@
           }
 
           &:hover:not(.modified) {
-            .rolling-stock {
-              opacity: 0.3;
+            .train-time {
+              color: var(--grey20)
             }
             .action-buttons {
               display: flex;
               z-index: 1;
             }
           }
+
+          &:hover:not(.selected, .mofied, .invalid) {
```

One typo. Also seems like it solves a bug, would you mind to put this in a separate commit ?
```suggestion
          &:hover:not(.selected, .modified, .invalid) {
```

---

Oupss, thank you

---

Done